mne/channels/layout.py

```diff
@@ -763,7 +766,7 @@ def plot(self, scale_factor=1.5, show_names=False):
                             show_names=show_names)
 
 
-def read_montage(kind, ch_names=None, path=None, scale=True):
+def read_montage(kind, ch_names=None, path=None, scale=None, transform=False):
```

Comment:
i guess this goes against a point I made earlier, but is there a case we would want to scale the points and not do a transform? I didn't think so earlier, but now I'm wondering if there's a case for it.